src/components/custom-field-input-text/custom-field-input-text.css

```diff
@@ -0,0 +1,84 @@
+@import '../../styles/colors.css';
+@import '../../styles/spacing.css';
+@import '../../styles/typography.css';
+
+.additional-information-icon {
+  border: 1px solid var(--palette-grey-light);
+  border-radius: 50%;
+  color: var(--palette-grey-dark);
+  cursor: pointer;
+  display: inline-block;
+  height: var(--spacing-large);
+  left: var(--spacing-medium);
+  padding-left: 1px;
+  position: relative;
```

Comment:
We want to avoid using relative positioning. While there are valid cases of relative positioning, we normally avoid it when we want to pixel-push around elements. 

position: relative && left: ?px

![Screen Shot 2019-12-05 at 5 46 09 PM](https://user-images.githubusercontent.com/841084/70288753-776edf00-1787-11ea-9724-e259c53b20d0.png)

position: static && margin-left: ?px

![Screen Shot 2019-12-05 at 5 46 29 PM](https://user-images.githubusercontent.com/841084/70288762-8190dd80-1787-11ea-9ea9-b560c86ac445.png)

Reply:
It's pretty hard to find this in the documentation but an answer in Quora does a good job at explaining the functional difference: 

https://www.quora.com/What-are-the-differences-between-margin-left-and-left-in-CSS